src/components/filterPatterns/popover/props.ts

```diff
@@ -4,9 +4,9 @@ import { ActionButtonProps } from '../ActionButton';
 export type PopoverFilterProps = FilterPatternProps &
   Omit<ActionButtonProps, 'onClose'> & {
     initialPopoverState?: 'open' | 'closed';
-
     onPopoverClose?: () => void;
     onPopoverOpen?: () => void;
     appliedLabel?: string;
     enforceHeight?: boolean;
+    height?: 'md' | 'lg';
```

Comment:
This API confuse me. If I see height as prop of PopoverFilter I would think it is hight of the popover itself.